app/services/csv_exporter.rb

```diff
@@ -35,18 +35,19 @@ def initialize(kase)
     @kase = kase
   end
 
-  def to_csv #rubocop:disable Metrics/CyclomaticComplexity, Metrics/MethodLength
+  def to_csv #rubocop:disable Metrics/MethodLength
```

Comment:
Given we are adding a field to a CSV, shouldn't there be a matching change in the CSV header?